Mint.css

```diff
@@ -0,0 +1,196 @@
+body {
+  margin: 0;
+  padding: 0;
+  background: #eceef1;
+  box-sizing: border-box;
```

Comment:
You are using this property in multiple places in css file, please follow the DRY rule (DON'T REPAET YOURSEFL). create a class with this property and apply it to HTML elements.
```
.someclass {
 box-sizing: border-box;
}
```

Reply:
ok done